### ISSUE TITLE
[REQUEST] Issues core conflict (mGBA GBA single core)

### ISSUE THREAD

**Issue by mrTruch** [KMFDManic/NESC-SNESC-Modifications#53]
Using two GAME BOY cores

Gambatte & mGBA there ara conflicts.

Even using the right command, maybe is related to my another bug bceause when I hit any GBC game it tuns black...when is a wrong command usually returns to main screen.

I would use gambatte for all own games (gb & gbc) and mGBA only for GBA just for compatbility and because I prefer GB gambatte colotization mode, almost as a SGB. heheh

So when I wrote the /bin/gambatte  or (bin/mgba) to specify it...Not working every time....

Most of GBC games take conflicts with mGBA, I don't know why...

So I thought maybe a mGBA GBA single core will be a great request. GBA ONLY

Thanks.

**Comment by KMFDManic** [comment]
/bin/gambatte and /bin/mgba still work, I just finished up recording the Dummies Guide for USB-HOST, and you will see a few things that may help you solve your issues.  I also included a few extra things in my Core Set relative to this stuff:)  I am also going to take a look at what you emailed me.  But, same as your main issue with USB-HOST, this issue is completely related to improper file names (internal and external), as well as incorrect desktop files.  Do not worry, you will better understand, once you see the video:)  I will also notate where you went wrong in what you emailed me!
